### ISSUE TITLE
Allow for user configuration of required metrics

### ISSUE THREAD

**Issue by bmistry12** [italux/sentry-prometheus-exporter#9]
**Is your feature request related to a problem? Please describe.**
Sometimes users may only require particular metrics or metrics for particular time frames from Sentry. With the current setup each time the `/metrics` endpoint is probed, there is a wait of `x` minutes (depends on project size) and all data is re-fetched from the API. In a situation where the probe is scraped multiple times a day or every `x_interval` minutes, data such as `number of project issues in the past 14d` is unlikely to be any different to the value x_interval minutes ago. 

For context, in the environment that this script is being tested on, a full run takes approximately 4m20s, and the goal is to reduce this time as much as possible by only fetching the required data each time - as an example just `project issues in the past 1h` rather than 24h + 14d. Usually our Prometheus scrapes from metric endpoints every 5-10 seconds, making this quite a big data time-lag.

tl;dr: Certain metrics aren't required by the end user, and so they are waiting `x` amount of time for redundant data when they can get the required data in up-to `x/4`.

**Describe the solution you'd like**
The user should be able to utilise environment variables or a config file to select which metrics they want to export data for. Only the specified data is scraped and returned from the API. By default all metrics should be scraped so that there is no issues or conflicts for users currently running the exporter. If possible there should be no extra configuration required for such users.

**Describe alternatives you've considered**
- Caching data that is unlikely to change for a set time period.
This is a viable option and will significantly reduce the runtime as per my end goal. However, taking the bigger picture into consideration this will not allow future users to only collect the metrics they actually want. Also, if data is corrupted/incorrect due to sentry being unreachable (for example), the data in the cache will be corrupted for the whole of that time period.

**Additional context**
In this setup, both Sentry & Prometheus are self-hosted and running in Kubernetes. Ultimately the environment and run method shouldn't matter, and the solution should be universal.
